src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,219 @@
+
+    private final TalonFX shoulderMainMotor = new TalonFX(Ports.ArmPorts.SHOULDER_MAIN_MOTOR);
+    private final TalonFX shoulderAuxMotor = new TalonFX(Ports.ArmPorts.SHOULDER_AUX_MOTOR);
+    private final CANCoder shoulderEncoder = new CANCoder(Ports.ArmPorts.SHOULDER_ENCODER);
+
+    private final TalonFX elbowMainMotor = new TalonFX(Ports.ArmPorts.ELBOW_MAIN_MOTOR);
+    private final TalonFX elbowAuxMotor = new TalonFX(Ports.ArmPorts.ELBOW_AUX_MOTOR);
+    private final CANCoder elbowEncoder = new CANCoder(Ports.ArmPorts.ELBOW_ENCODER);
+
+    private final UnitModel unitModel = new UnitModel(ArmConstants.TICKS_PER_RADIAN);
+    private double shoulderFeedforward;
+    private double elbowFeedForward;
+    private double shoulderSetpoint;
+    private double elbowSetpoint;
+
+    private Arm() {
+        super(new ArmInputsAutoLogged());
+        configureMainMotor(shoulderMainMotor, ArmConstants.shoulderP, ArmConstants.shoulderI, ArmConstants.shoulderD, shoulderEncoder);
+        configureAuxMotor(shoulderAuxMotor, shoulderMainMotor);
+
+        configureMainMotor(elbowMainMotor, ArmConstants.elbowP, ArmConstants.elbowI, ArmConstants.elbowD, elbowEncoder);
+        configureAuxMotor(elbowAuxMotor, elbowMainMotor);
+    }
+
+    /**
+     * Get the instance of the arm subsystem.
+     * @return Arm instance
+     */
+    public static Arm getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Arm();
+        }
+        return INSTANCE;
+    }
+
+    /**
+     * Configures the aux motors
+     * @param auxMotor
+     * @param mainMotor
+     */
+    private void configureAuxMotor(TalonFX auxMotor, TalonFX mainMotor) {
+        auxMotor.follow(mainMotor);
+        auxMotor.enableVoltageCompensation(ArmConstants.ENABLE_VOLT_COMPENSATION);
+        auxMotor.configVoltageCompSaturation(ArmConstants.VOLT_COMP_SATURATION);
+        auxMotor.configMotionAcceleration(ArmConstants.MOTION_ACCELERATION);
+        auxMotor.configMotionCruiseVelocity(ArmConstants.MOTION_CRUISE_VELOCITY);
+        auxMotor.setNeutralMode(NeutralMode.Brake);
+    }
+
+    /**
+     * Configures the main motors
+     * @param mainMotor
+     * @param kP
+     * @param kI
+     * @param kD
+     * @param encoder
+     */
+    private void configureMainMotor(TalonFX mainMotor, double kP, double kI, double kD, CANCoder encoder) {
+        mainMotor.enableVoltageCompensation(ArmConstants.ENABLE_VOLT_COMPENSATION);
+        mainMotor.configVoltageCompSaturation(ArmConstants.VOLT_COMP_SATURATION);
+        mainMotor.setNeutralMode(NeutralMode.Brake);
+        mainMotor.setInverted(ArmConstants.CLOCKWISE);
+
+        mainMotor.config_kP(0, kP);
+        mainMotor.config_kI(0, kI);
+        mainMotor.config_kD(0, kD);
+
+        mainMotor.configNeutralDeadband(ArmConstants.DEADBAND);
+        mainMotor.setSelectedSensorPosition(unitModel.toTicks(encoder.getAbsolutePosition()));
```

Comment:
This line runs once for each motor. You want the Talon FX to continuously read measurements from the CANCoder.